projects/step-core/src/lib/services/view-registry.service.ts

```diff
@@ -279,7 +290,7 @@ export class ViewRegistryService implements OnDestroy {
   }
 
   registerMenuEntry(title: string, id: string, icon: string, options?: { weight?: number; parentId?: string }): void {
-    if (!id || this.registeredMenuIds.includes(id)) {
+    if (!id) {
```

Comment:
I'm pretty sure that we can't remove this as it might produce duplicated menu entries. We have some plugins that insert the same entry as they both require it and don't have awareness of each other. 

As the title and the id can be separate this should still work for bookmarks of the same name if they have different ids, correct?

Reply:
It works for bookmarks with the same name, but it doesn't work for any bookmark that has the same id (and also route in this case) as any entry from sidebar menu